packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -236,6 +237,47 @@ const assetFiles = (name, tailwind) => {
   return fileList
 }
 
+const formatters = (name) => {
+  const outputPath = path.join(getPaths().web.src, 'lib', 'formatters.tsx')
+  const outputPathTest = path.join(
+    getPaths().web.src,
+    'lib',
+    'formatters.test.tsx'
+  )
+
+  // skip files that already exist on disk, never worry about overwriting
+  if (fs.existsSync(outputPath)) {
```

Comment:
Shouldn't we override this when the `--force` flag is passed?

Reply:
I chose to treat the formatters file the same way we treat the common css file we have for scaffolds. (You can see the code for that on line 220ish)

Reply:
Yeah totally weird behaviour IMO. But hey atleast it's consistent.